Components/Typography/index.html

```diff
@@ -0,0 +1,147 @@
+          <li><a href="../Avatar/">Avatars</a></li>
+          <li><a href="../Badge/">Badges</a></li>
+          <li><a href="../Buttons/">Buttons</a></li>
+          <li><a href="../Cards/">Cards</a></li>
+          <li><a href="../Grid/">Grids</a></li>
+          <li><a href="../Images/">Images</a></li>
+          <li><a href="../Input/">Input</a></li>
+          <li><a href="../Lists/">Lists</a></li>
+          <li><a href="../Modal/">Modal</a></li>
+          <li><a href="../Navigation/">Navigation</a></li>
+          <li><a href="../Rating/">Rating</a></li>
+          <li><a href="../Slider/">Slider</a></li>
+          <li><a class="active" href="#">Typography</a></li>
+        </ul>
+      </aside>
+      <main>
+        <section class="m-2">
+          <h1 class="component-name">Typography</h1>
+          <p>
+            Typographies are utility classes used for maintaining a uniform text
+            stylying across the website.
+          </p>
+        </section>
+        <section class="m-2">
+          <h2>Headings utility</h2>
+          <p>
+            Heading utilities provide the standard font size for all headings.
+          </p>
+          <div class="mb-3 px-1 border">
+            <h1 class="my-1">Heading 1</h1>
+            <h2 class="my-1">Heading 2</h2>
+            <h3 class="my-1">Heading 3</h3>
+            <h4 class="my-1">Heading 4</h4>
+            <h5 class="my-1">Heading 5</h5>
+            <h6 class="my-1">Heading 6</h6>
+          </div>
+          <h2>Text alignments</h2>
+          <p>
+            Text alignments provide the alignment properties for text.
+          </p>
+          <div class="mb-3 p-1 border">
+            <p class="text-left">Left aligned Text</p>
+            <p class="text-center">Center aligned Text</p>
+            <p class="text-right">Right aligned Text</p>
+          </div>
+          <h2>Coloured Text</h2>
+          <p>
+            Coloured text utility provide various color options for text.</p>
+          <div class="mb-3 p-1 border">
+            <p class="primary-text">Primary Text</p>
+            <p class="secondary-text">Secondary Text</p>
+            <p class="success">Success Text</p>
+            <p class="warning">Warning Text</p>
+            <p class="error">Error Text</p>
+            <p class="grey-text">Grey Text</p>
+          </div>
+          <p class="text-center">
+            Refer the below code to use the Typography classes. Make changes
+            as per your requirements.
+          </p>
+          <div class="code-block">
+            <script src=""></script>
+          </div>
+        </section>
+      </main>
+    </div>
+    <footer class="footer mb-1">
+      Made by
+      <a href="https://github.com/TejasShekar" target="_blank">
+        <span class="span">Tejas</span>
+      </a>
+    </footer>
+    <script src="../../Stylesheets/theme-toggle.js"></script>
+    <script src="./modal.js"></script>
```

Comment:
You don't need a modal in typography.

Reply:
Thanks for pointing out. Will remove it.